blueprint/src/web.tex

```diff
@@ -0,0 +1,31 @@
+% This file makes a web version of the blueprint
+% It should include all the \usepackage needed for this version.
+% The template includes standard AMS packages.
+% It is otherwise a very minimal preamble (you should probably at least
+% add cleveref and tikz-cd).
+
+\documentclass{report}
+
+\usepackage{amsmath,amsfonts,amsthm,amssymb,hyperref,graphicx, bbold}
+%\usepackage[bb=stix]{mathalpha}
+\DeclareGraphicsExtensions{.svg,.png,.jpg}
+\usepackage[capitalize]{cleveref}
+\usepackage[showmore, dep_graph]{blueprint}
+
+\usepackage{tikz-cd}
+\usepackage{tikz}
+\input{macros/common}
+\input{macros/web}
+
+\home{https://MAO3J1m0Op.github.io/algebra-in-lean}
+\github{https://github.com/MAO3J1m0Op/algebra-in-lean}
+\dochome{https://MAO3J1m0Op.github.io/algebra-in-lean/docs}
+
+\title{Algebra in Lean}
+\author{Will Harris, Justin Morrill, Dominic King, Adam Kern, Clara Henne, Arim Lim, 
+William Ho, Stavan Jain, Anoushka Sinha, Ricardo Prado Cunha}
```

Comment:
This should be in sync with `print.tex`. Maybe put this stuff in `\input{macros/common}`? Not sure if there's a "correct" place to put it.